text/0847-deprecate-support-for-ember-cli-qunit-and-ember-cli-mocha-when-generating-test-blueprints.md

```diff
@@ -0,0 +1,114 @@
+---
+stage: accepted
+start-date: 2022-08-26T00:00:00.000Z
+release-date:
+release-versions:
+teams: # delete teams that aren't relevant
+  - cli
+  - data
+  - framework
```

Comment:
This needs to be updated, right?

Reply:
I believe this list is correct in this case. There are blueprints in both data and cli that will be affected.

Reply:
confirm, which is why I shipped it. Consider my review to be data's ack that blueprint changes are to be made.

Reply:
👍